pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -205,6 +220,11 @@ type Framework interface {
 	// cycle is aborted.
 	RunPrefilterPlugins(pc *PluginContext, pod *v1.Pod) *Status
 
+	// RunFilterPlugins runs the set of configured filter plugins for pod on each
```

Comment:
"each candidate host" -> "the given node"

"If any of these plugins returns error" -> "If any of these plugins returns any status other than 'Success'"

Reply:
It has been addressed in the latest code.